go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/emersion/go-textwrapper v0.0.0-20200911093747-65d896831594
 	github.com/martinlindhe/base36 v1.1.0
-	github.com/stretchr/testify v1.3.0 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/stretchr/testify v1.3.0
```

Comment:
Why do you pull in testify?
go-message doesn't use a testing framework, why introduce it now?

Reply:
I use it automatically as it saves a lot of boilerplate and prevents a lot of bugs. But I will remove it.